andela/ah-frontend-athena
# #162163249 Reading stats of all users.

### What does this PR do?
This PR ensures that read and views count per article is recorded when the article is opened. The statistics( reads and views) of the articles are displayed to the author of the article.
#### Description of Task to be completed?

- Record both read time and view time of an article.

- Display the reads and views to the author of the article.

#### How should this be manually tested?

- Create an article.

- Enter go to your profile page and click the button `Your read statistics`

- The article will have 0 reads and 0 views.

- Open the article and read it.

- Go back and check on your reading statistics, the views and/or reads values will have increased depending on how much time you spent on the article.

#### What are the relevant pivotal tracker stories?
[Delivers #162163249]
#### Screenshots:
![image](https://user-images.githubusercontent.com/40671517/52068401-a9bf1700-258d-11e9-8897-89e5d2240191.png)
Screenshot of newly created article
![image](https://user-images.githubusercontent.com/40671517/52068549-ef7bdf80-258d-11e9-8227-3ce5687c2eb2.png)

Click the lower button
![image](https://user-images.githubusercontent.com/40671517/52068603-07536380-258e-11e9-876f-e2f95f44a55c.png)
View read stats of the article.
![image](https://user-images.githubusercontent.com/40671517/52068709-39fd5c00-258e-11e9-8334-6f0f54f7bc6f.png)
Read stats once the article has been viewed
